phoenix-core/src/main/java/org/apache/phoenix/iterate/ExplainTable.java

```diff
@@ -178,6 +180,10 @@ protected void explain(String prefix, List<String> planSteps,
                 }
             } while (filterIterator.hasNext());
         }
+        Set<PColumn> dataColumns = context.getDataColumns();
```

Comment:
Technically if you wanted to be pedantic this would be after SERVER FILTER BY FIRST KEY ONLY and before any other filters. Since Phoenix puts these together in one line I chose to put this in front.